exist-core/src/test/xquery/dates/convert-dates.xqm

```diff
@@ -25,68 +25,68 @@ module namespace fd="http://exist-db.org/xquery/test/convert-dates";
 
 declare namespace test="http://exist-db.org/xquery/xqsuite";
 
-(: verify that we can convert xs:string to :xs:dateTimeStamp  :)
+(: verify that we can convert xs:string to :xs:dateTime :)
 declare
     %test:args("2012-06-26T23:14:22.566+02:00")
     %test:assertEquals("11:14 pm on Tuesday, June 26th, 2012")
-function fd:format-dateTimeStamp($date as xs:dateTimeStamp) {
+function fd:format-dateTime($date as xs:dateTime) {
     format-dateTime($date, "[h00]:[m00] [P] on [FNn], [MNn] [D1o], [Y]", "en", (), ())
 };
 
-(: verify that we can't convert xs:string without timezone to :xs:dateTimeStamp  :)
+(: verify that we can't convert xs:string without timezone to :xs:dateTime :)
 declare
     %test:args("2012-06-26T23:14:22.566")
     %test:assertError
-function fd:convert-date-time-stamp($date as xs:dateTimeStamp) {
+function fd:convert-date-time($date as xs:dateTime) {
```

Comment:
I think this test only made sense in its original form before this change, since xs:dateTimeStamp is the datatype that requires an explicit timezone offset - see https://www.w3.org/TR/xmlschema11-2/#dateTimeStamp.

Reply:
This has also now been undone.